ChangeLog

```diff
@@ -16,16 +16,42 @@ Release date: TBA
   Closes PyCQA/pylint#7488.
 
 
-What's New in astroid 2.12.12?
+What's New in astroid 2.12.13?
 ==============================
 Release date: TBA
 
 
 
+What's New in astroid 2.12.12?
+==============================
+Release date: 2022-10-19
+
+* Add the ``length`` parameter to ``hash.digest`` & ``hash.hexdigest`` in the ``hashlib`` brain.
+
+  Refs PyCQA/pylint#4039
+
+* Prevent a crash when a module's ``__path__`` attribute is unexpectedly missing.
+
+  Refs PyCQA/pylint#7592
+
+* Fix inferring attributes with empty annotation assignments if parent
+  class contains valid assignment.
+
+  Refs PyCQA/pylint#7631
+
+
 What's New in astroid 2.12.11?
 ==============================
 Release date: 2022-10-10
 
+* Improve detection of namespace packages for the modules with ``__spec__`` set to None.
+
+  Closes PyCQA/pylint#7488.
+
+* Fixed a regression in the creation of the ``__init__`` of dataclasses with
```

Comment:
Seems like this was still in the wrong release and thus not in the changelog?

Reply:
I used the release note from GitHub as a source of truth to fix the release note, if they were wrong then this is also wrong. (I had a conflict when cherry-picking the fix from main)